### PR TITLE
Add 'email on updated' to notifications config

### PR DESCRIPTION
**Pull request by vboctor**
Fixes #20322

### CODE REVIEW

File: core/email_api.php [hunk 1]
```diff
@@ -1555,3 +1555,29 @@ function email_shutdown_function() {
...
+ * @return array List of actions
+ */
+function email_get_actions() {
+	$t_actions = array( 'updated', 'owner', 'reopened', 'deleted', 'bugnote' );
+
+	if( config_get( 'enable_sponsorship' ) == ON ) {
+		$t_actions[] = 'sponsor';
+	}
+
+	$t_actions[] = 'relation';
```
Comment (dregad) [inline core/email_api.php]: Any particular reason why this is not in the initial array ?

File: core/email_api.php [hunk 2]
```diff
@@ -1555,3 +1555,29 @@ function email_shutdown_function() {
...
+
+	$t_statuses = MantisEnum::getAssocArrayIndexedByValues( config_get( 'status_enum_string' ) );
+	ksort( $t_statuses );
+	reset( $t_statuses );
+
+	foreach( $t_statuses as $t_status => $t_label ) {
```
Comment (dregad) [inline core/email_api.php]: No need for `$t_status =>` if you're not using it

**Comment by dregad** [comment]
+1 with minor comments

Good call moving redundant code to dedicated email_get_actions() API.

File: core/email_api.php (hunk 1 above)
Comment (vboctor) [inline core/email_api.php]: Nope, this was the case for the original code, I'll combine it with the initial array initialization.

File: core/email_api.php (hunk 2 above)
Comment (vboctor) [inline core/email_api.php]: Yep, I removed it.  It was used in one case of the old code and not in the other.  I generally don't mind including it for readability, but will remove it.

**Pull request merged**